### ISSUE TITLE
Switch to keras import

### ISSUE THREAD

**Issue by fpolachowski** [psorus/eereid#16]
With Keras 3.0 keras can be directly imported.
Instead of:
`from tensorflow import keras`
Now supports:
`import keras`

**Comment by psorus** [comment]
true. But depends on the environment. I think changing this would mean that you actually have to install keras. And I mean there is no benefit right?
